src/App.scss

```diff
@@ -69,44 +68,44 @@ html {
     border: 0px solid;
     color: black;
   }
-  
-  
+
+
   /** CHECKBOX*/
   /* Original styles by Shaw : https://codepen.io/shshaw/pen/WXMdwE */
   $duration: 0.4s;
   $stroke: #42d7e4;
   $colorText: #076b74;
   $colorTextChecked: #a5e7ec;
-  
+
   .todo {
     display: block;
     position: relative;
     padding: 1em 1em 1em 16%;
     margin: 0 auto;
-    cursor: pointer; 
-    border-bottom: solid 1px #ddd; 
+    cursor: pointer;
+    border-bottom: solid 1px #ddd;
     &:last-child { border-bottom: none; }
   }
-  
+
   .todo.new-item {
     opacity: 0;
     transform: translateX(100px);
     animation: fadeIn .3s linear forwards;
   }
-  
+
   .todo__state {
     position: absolute;
     top: 0;
     left: 0;
     opacity: 0;
   }
-  
+
   .todo__text {
     color: $colorText;
     text-align: left;
-    transition: all $duration/2 linear $duration/2;
+    transition: all calc($duration / 2) linear calc($duration / 2);
```

Comment:
this was causing a warning in the console

![Screenshot 2024-09-18 at 18 38 37](https://github.com/user-attachments/assets/6b8b6ffd-a49f-4495-af98-2ce0087e60d5)